### ISSUE TITLE
Igniter: Add `plug :load_from_bearer` to `:graphql` pipeline if it exists in router

### ISSUE THREAD

**Issue by sevenseacat** [team-alembic/ash_authentication_phoenix#523]
The current igniter will augment the `:api` pipeline (added by AshJsonApi) to load the user from the the authorization header, but not the `:graphql` pipeline (added by AshGraphql).

It would probably be good to add it to both!

I think it's currently a bit more complicated than adding another line here https://github.com/team-alembic/ash_authentication_phoenix/blob/main/lib/mix/tasks/ash_authentication_phoenix.install.ex#L110 because this looks like it will create the pipeline if it doesn't exist?

**Comment by sevenseacat** [comment]
The `set_actor` plug should also be added to the GraphQL pipeline. ie.

```elixir
  import AshAuthentication.Plug.Helpers

  pipeline :graphql do
    plug :load_from_bearer
    plug :set_actor, :user
    plug AshGraphql.Plug
  end
```

**Comment by zachdaniel** [comment]
Interesting..doesn't `set_actor` need to be done for both then?

**Comment by zachdaniel** [comment]
Resolved in 15f191046e12d853eaa648ca212e5a5b1705b593
